src/Migrator.php

```diff
@@ -234,20 +241,31 @@ private function ensureEmptyTargetTable($table, Connection $sourceConnection, Co
             $createSqlRow = $sourceConnection->query('SHOW CREATE TABLE ' . $sourceConnection->quoteIdentifier($table))
                 ->fetch(\PDO::FETCH_ASSOC);
             $createSql = $createSqlRow['Create Table'];
-            $triggerSql = $this->generateTableTriggerSql($table, $sourceConnection);
+            $this->generateTableTriggerSql($table, $sourceConnection);
         } elseif ($driverName === 'pdo_sqlite') {
             $schemaSql = 'SELECT sql FROM sqlite_master WHERE type="table" AND tbl_name=' . $sourceConnection->quoteIdentifier($table);
             $createSql = $sourceConnection->query($schemaSql)->fetchColumn();
-            $triggerSql = $this->generateTableTriggerSql($table, $sourceConnection);
+            $this->generateTableTriggerSql($table, $sourceConnection);
```

Comment:
Generating the trigger sql probably doesn't belong in a method called `ensureEmptyTargetTable`?

Reply:
Refactored to keep all the trigger logic together in one place